src/components/sequencing/Sequences.svelte

```diff
@@ -59,6 +63,41 @@
     const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
     goto(`${base}/sequencing/new${workspaceId ? `?${SearchParameters.WORKSPACE_ID}=${workspaceId}` : ''}`);
   }
+
+  async function importLibarySequences(): Promise<void> {
+    const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
+    if (workspaceId === null) {
+      console.log("Workspace doesn't exist");
+      showFailureToast("Library Import: Workspace doesn't exist");
+      return;
+    }
+    const { confirm, value } = await showLibrarySequenceModel();
+
+    if (confirm && value) {
+      const fileName = await value.libraryFile.name;
+      const type = fileName.slice(fileName.lastIndexOf('.') + 1);
+      if (type !== 'satf') {
+        console.log(`Unsupported file type ${type}`);
```

Comment:
Do we need to log here if we're already showing the toast?

Reply:
Nope, just for debugging. I removed it.